tests/README.md

```diff
@@ -36,11 +36,17 @@ cd packages/ckeditor5-dev-tests
 yarn link
 ```
 
+After that, install the CKEditor 5 testing environment:
+
+```sh
+npm i --save-dev @ckeditor/ckeditor5-dev-tests
```

Comment:
This step doesn't work for me, it produces error 🤔

Reply:
I was able to run the code above but even so it would still result in errors later on caused by missing `istanbul-instrumenter-loader`. The only way to resolve this alltogether was to include the `istanbul-instrumenter-loader` as a devDependency and skip the steps highlighted by you, so they were removed.

Reply:
I see I was not the first one to encounter this problem - https://github.com/ckeditor/ckeditor5/issues/7201.